### ISSUE TITLE
Release on PyPI

### ISSUE THREAD

**Issue by hassankhan** [modesttree/Projeny#32]
Once the project is updated to work on OSX, then installation (should) be as simple as `pip install projeny`. IIRC it would require someone to register the package on PyPI.

**Comment by svermeulen** [comment]
Yeah.  I'd also like to add it to chocolatey to make it easier for windows users.

**Comment by hassankhan** [comment]
Wouldn't adding it to PyPI solve that problem as well though?

**Comment by svermeulen** [comment]
Yeah, you could use `pip install projeny` on windows as well, but some people might not have python or they might not want to run it from source.  So chocolatey would install/update just the compiled exe for projeny
